homeassistant/components/bond/utils.py

```diff
@@ -42,6 +42,20 @@ def supports_direction(self) -> bool:
             > 0
         )
 
+    def supports_light(self) -> bool:
+        """Return True if this device supports any of the light related commands."""
+        actions: List[str] = self._attrs["actions"]
+        return (
+            len(
```

Comment:
Using `bool` also works and is a bit shorter than checking the length of the list.

```py
return bool([...])
```

Reply:
thanks, fixed this here: https://github.com/home-assistant/core/pull/37821